### PR TITLE
Disable TPM2 check on SLE < 15-SP4

### PR DESCRIPTION
**Pull request by paolostivanin**
Due to https://bugzilla.suse.com/show_bug.cgi?id=1199864, checking the `binary_bios_measurements` will fail on SLES <= 15-SP3.

Verification runs:
- 15-SP3: https://openqa.suse.de/tests/8816297
- 15-SP2: not needed since tpm2 is not available there

### CODE REVIEW

**Comment by Amrysliu** [comment]
The code looks good to me, but please check the tidy-check failure.

File: lib/swtpmtest.pm [hunk 1]
```diff
@@ -111,9 +112,12 @@ sub swtpm_verify {
         assert_script_run("grep '0 :' $result_file");    # The "tpm2_pcrread" command will show sha1:0 value
     }
 
-    # Measured boot check
-    # If measured boot works fine, it can record available algorithms and pcrs
-    assert_script_run("cat $result_file | egrep 'AlgorithmId|pcrs'");
+    # Due to bsc#1199864, the following works only on SLE >= 15-SP4
+    if (!is_sle('<15-SP4')) {
```
[the remark this answers is not in the document]
Comment (paolostivanin) [reply lib/swtpmtest.pm:116]: `!is_sle('<15-SP4')` includes also opensuse

**Comment by paolostivanin** [comment]
@Amrysliu I've just executed `make tidy-check` and didn't get any error. What should I do?

File: lib/swtpmtest.pm (hunk 1 above)
[the remark this answers is not in the document]
Comment (rfan1) [reply lib/swtpmtest.pm:116]: > if (is_sle('>=15-SP4')) It is better to express, I think.

We should consider opensuse as well

**Review by Amrysliu — APPROVED** [review]

**Pull request merged**